### ISSUE TITLE
Translations API group buildProject method returns error

### ISSUE THREAD

**Issue by AlexanderZanin** [crowdin/crowdin-api-client-js#77]
When I try to call translationsApi.buildProject(projectId) method it returns following error: 
`{ error: { message: 'Unsupported Content-Type Header', code: 415 } }`

According to the [documentation](https://support.crowdin.com/api/v2/#tag/Translations/paths/~1projects~1{projectId}~1translations~1builds/post) the Content-Type Header should be set to `application/json`

How can I customise the request with the proper Content-Type Header? 

This is how I create the instance: 
```
const { Translations } = require('@crowdin/crowdin-api-client');

const translationsApi = new Translations(
      {
        token,
      },
      {
        headers: {
          'Content-Type': 'application/json',
        },
      },
    );
```
This approach doesn't work.

When I checked the source code I didn't see the way to extend the defaultConfig that's used under the hood.

**Comment by DimaYashchyshyn** [comment]
Hi @AlexanderZanin 
From first glance, it seems you're using header we don't support 
Can you provide us with the action you want to do or header you are want to add
We'll try to figure out how to safely extend it

**Comment by AlexanderZanin** [comment]
Hi @DimaYashchyshyn! If I add this header directly to the source code to this line: https://github.com/crowdin/crowdin-api-client-js/blob/master/src/core/index.ts#L173 everything works.

I just use this method: https://github.com/crowdin/crowdin-api-client-js/blob/master/src/translations/index.ts#L63
And it seems to be broken by default because of the header that is not set.

If I create the instance in this way: 
`const { Translations } = require('@crowdin/crowdin-api-client');

`const translationsApi = new Translations({
token,
});`
The result is the same with the same error.

**Comment by DimaYashchyshyn** [comment]
@AlexanderZanin 
I got you. We'll reach you tomorrow with some updates

**Comment by AlexanderZanin** [comment]
@DimaYashchyshyn, great! Thank you a lot!

**Comment by AlexanderZanin** [comment]
Hi!
Any updates on this?

**Comment by yevheniyJ** [comment]
@AlexanderZanin Hi there! I am on track with this. Hopefully soon new version will be released with a fix for it.

**Comment by yevheniyJ** [comment]
Fixed in v1.8.14 ^^
